src/containers/Sync/Sync.jsx

```diff
@@ -91,7 +94,9 @@ class Sync extends Component {
     if (this.props.syncStep === 'finished') {
       return this.props.onClose()
     }
-
+    if (this.props.syncStep === 'error') {
```

Comment:
I think we should let the modal opened in order to display the error to the user ?

Reply:
The modal is opened, the purpose of the if statement is only to not execute the code above.